index.html

```diff
@@ -4,9 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Anwar Hussaini's Portfolio</title>
   </head>
   <body>
-    <h1 class="heading-primary">My Portfolio</h1>
+    <header class="main-header bg-color">
+      <div class="notification-bar">
+        <img src="Light.png" alt="Notification Bar" class="notification-img" />
+      </div>
+      <div class="container">
+        <nav class="main-nav">
+          <div class="logo-box"></div>
+          <div class="menus-box"><i class="fa-solid fa-bars menu icon"></i></div>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="section-hero bg-color">
+        <div class="container">
+          <h1 class="heading-primary">
+            <div class="hero-name">
+              <span>Hey There.</span>
+              <span>I'm Anwar</span>
+            </div>
+            <div class="hero-job">
+              <span>I'm a software</span>
+              <span>developer</span>
+            </div>
+          </h1>
+
+          <div class="hero-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </div>
+
+          <div class="social-media-icons">
+            <ul class="icons">
+              <li><i class="fa-brands fa-twitter icon"></i></li>
+              <li>
+                <i class="fa-brands fa-linkedin-in icon"></i>
+              </li>
+              <li><i class="fa-solid fa-m icon"></i></li>
+              <li><i class="fa-brands fa-github icon"></i></li>
+              <li><i class="fa-brands fa-angellist icon"></i></li>
+            </ul>
+          </div>
+        </div>
+        <div class="navigator">
+          <i class="fa-solid fa-angle-down"></i>
+        </div>
```

Comment:
- [ ] As I mentioned above :point_up_2: please stick to the Figma design your navigator section is not the same as the Figma design kindly fix it :slightly_smiling_face: check below :point_down: the comparison between your design and Figma design :+1: 

| Your Design| Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-27 15-55-25](https://user-images.githubusercontent.com/45226848/234848678-f4063348-98f9-4d1a-9dc2-49640586ee85.png) | ![Screenshot from 2023-04-27 15-58-55](https://user-images.githubusercontent.com/45226848/234848703-7ee54a83-a660-4f3e-8e7d-bab4b5fe4f7a.png) |